payment-handler/can-make-payment-event.https.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<meta charset="utf-8">
+<title>Tests for CanMakePaymentEvent</title>
+<link rel="help" href="https://w3c.github.io/payment-handler/#the-canmakepaymentevent">
+<link rel="manifest" href="manifest.json">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+async function registerAppAndBuildPaymentRequest(methodName) {
+  await navigator.serviceWorker.register('app-can-make-payment.js');
+  registration = await navigator.serviceWorker.ready;
+  await registration.paymentManager.instruments.set('instrument-key', {
+    name: 'Test Payment Method',
+    enabledMethods: [methodName],
+  });
+  const request = new PaymentRequest([{
+    supportedMethods: methodName,
+  }], {
+    total: {
+      label: 'Total',
+      amount: {
+        currency: 'USD',
+        value: '0',
+      },
+    },
+  });
+  return request;
+}
+
+promise_test(async t => {
+  const request = await registerAppAndBuildPaymentRequest(window.location.origin + '/canMakePayment-false');
```

Comment:
nit: would you mind putting the url into a variable. It helps a lot with debugging. 

```JS
const url = window.location.origin + '/canMakePayment-false'; 
```